expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResource.java

```diff
@@ -216,14 +219,24 @@ private static void addJoinFieldWhenAnthology(ObjectNode sortedJson, Reference r
         }
     }
 
+    private static boolean instanceTypeCanBeAnthology(PublicationInstance<?> instanceType) {
+        return instanceType instanceof BookMonograph
+                || instanceType instanceof ReportBasic
+                || instanceType instanceof BookAnthology
+                || instanceType instanceof ReportBookOfAbstract;
+    }
+
+    private static boolean instanceTypeCanBePartOfAnthology(PublicationInstance<?> instanceType) {
+        return instanceType instanceof ChapterArticle
+               || instanceType instanceof ChapterConferenceAbstract
+               || instanceType instanceof ChapterInReport;
+    }
```

Comment:
Why not to use publicationContext type when we are checking if resource can be an Anthology?
Instead of listing all the possible "subtypes", we can check "main type".

![image](https://github.com/user-attachments/assets/4e02c4cb-23fb-476f-be68-0e7ed9497f4a)